src/ui/layout/mct-tree.vue

```diff
@@ -148,549 +125,379 @@ export default {
             if (this.abortController) {
                 this.abortController.abort();
                 delete this.abortController;
             }
 
-            this.searchValue = '';
-
             if (!this.openmct.router.path) {
                 return;
             }
 
-            if (this.currentPathIsActivePath() && !this.isLoading) {
-                this.skipScroll = false;
-                this.autoScroll();
-
-                return;
-            }
-
-            let routePath = [...this.openmct.router.path];
-            routePath.shift(); // remove the child, so it navigates to the parent
-            routePath.push(this.root); // adding root
-            this.beginNavigationRequest('jumpTo', [...routePath].reverse());
+            this.showCurrentPathInTree();
+        },
+        resetTreeNavigation() {
+            [...this.openTreeItems].reverse().map(this.closeTreeItem);
         },
         searchValue() {
             if (this.searchValue !== '' && !this.activeSearch) {
                 this.activeSearch = true;
                 this.$refs.scrollable.scrollTop = 0;
-                this.skipScroll = true;
             } else if (this.searchValue === '') {
                 this.activeSearch = false;
-                this.skipScroll = false;
             }
         },
-        ancestors() {
-            this.observeAncestors();
-        },
-        availableContainerHeight() {
+        mainTreeHeight() {
             this.updateVisibleItems();
         },
         focusedItems() {
             this.updateVisibleItems();
+        },
+        openTreeItems() {
+            this.setSavedOpenItems();
         }
     },
     async mounted() {
-        this.isLoading = true;
-
         await this.initialize();
+        await this.loadRoot();
+        this.isLoading = false;
 
-        let rootComposition = await this.loadRoot();
-        let path = ROOT_PATH;
-
-        if (!rootComposition) {
-            return;
-        }
-
-        if (!this.multipleRootChildren && rootComposition[0]) {
-            let id = this.openmct.objects.makeKeyString(rootComposition[0].identifier);
-            path += '/' + id;
-        }
-
-        this.beginNavigationRequest('jumpTo', path);
+        await this.syncTreeOpenItems();
     },
     created() {
         this.getSearchResults = _.debounce(this.getSearchResults, 400);
         this.handleWindowResize = _.debounce(this.handleWindowResize, 500);
     },
     destroyed() {
         window.removeEventListener('resize', this.handleWindowResize);
-        this.stopObservingAncestors();
     },
     methods: {
         async initialize() {
-            // required to index tree objects that do not have search providers
+            this.isLoading = true;
             this.openmct.$injector.get('searchService');
-
+            this.getSavedOpenItems();
             window.addEventListener('resize', this.handleWindowResize);
 
             await this.calculateHeights();
 
             return;
```

Comment:
Yeah, let's ask @akhenry I thought we wanted this back in now that we could have multiple open. Removing this would reset the tree every time you load.

Reply:
If that's what you're even referring to : )